ls1intum/Artemis
# `Development`: Persist logs for 90 instead of 7 days.

### Checklist
#### General



### Motivation and Context
Currently the log files are persisted for 7 days: e.g. today on August 29, the logfile for the day is present as well as 7 archives with gzipped-logs (one for each of the last 7 days).

We want to keep the logs longer to enable further analysis of issues/performance.

### Description
Change the max history to 90 (see the [documentation](https://logback.qos.ch/manual/appenders.html#tbrpMaxHistory)).

### Steps for Testing
Due to the nature of this change, it can only be tested locally.
The testing is performed by creating files that should be removed and files that should be persisted and checking whether the correct files are removed.

1. Set the following temporary values in your config:
1.1 `logging.logback.rollingpolicy.clean-history-on-start: true`: Usually the cleanup is only done when rolling over the logfile (= on midnight), but for testing purposes, it is easier, if the cleanup is performed on the startup of the application
1.2 `logging.file.name`: `/some/path/on/your/local/filesystem/artemis.log`: This specifies the path for the file of the current day. Choose a location that you can access on your local filesystem
2. Create some files that should be persisted (=files that have a date of the last 90 days): ` touch /some/path/on/your/local/filesystem/artemis.log.2022-08-27.0.gz  /some/path/on/your/local/filesystem/artemis.log.2022-07-27.0.gz`
3. Create some files that should be removed (=older than 90 days): `touch /some/path/on/your/local/filesystem/artemis.log.2022-05-27.0.gz  /some/path/on/your/local/filesystem/artemis.log.2022-05-22.0.gz`
4. Start the server and ensure that the correct files are removed


Prerequisites:

### Review Progress





#### Code Review
#### Manual Tests
